console/services/app_config/port_service.py

```diff
@@ -64,7 +64,9 @@ def check_k8s_service_name(tenant_id, k8s_service_name, container_port=None):
         # make k8s_service_name unique
         try:
             port = port_repo.get_by_k8s_service_name(tenant_id, k8s_service_name)
-            if container_port is None or port.container_port != container_port:
+            if not component_id or not container_port:
+                raise ErrK8sServiceNameExists
+            if port.service_id != component_id or port.container_port != container_port:
```

Comment:
should use `and`?

Reply:
It should be `or`. If one of the conditions is not met, it means that it is not the same port and an exception should be raise